ipywebrtc/webrtc.py

```diff
@@ -1,20 +1,30 @@
 from __future__ import absolute_import
-import ipywidgets as widgets
-import ipywidgets
-from traitlets import Unicode, Integer
-import traitlets
+import os
 import logging
+try:
+    from urllib import urlopen  # py2
+except ImportError:
+    from urllib.request import urlopen  # py3
+from traitlets import (
+    Bool, Bytes, Dict, Instance, List, Unicode
+)
+from ipywidgets import DOMWidget, register, widget_serialization
+from ipython_genutils.py3compat import string_types
+import ipywebrtc._version
 
 logger = logging.getLogger("jupyter-webrtc")
-
-import ipywebrtc._version
 semver_range_frontend = "~" + ipywebrtc._version.__version_js__
 
-class HasStream:
-    pass # to indicate it has a .stream property on the JS side
-@widgets.register('webrtc.MediaStream')
-class MediaStream(widgets.DOMWidget, HasStream):
-    """Represents a media source."""
+
+@register('webrtc.MediaStream')
```

Comment:
We can drop the string, and drop widget 6 support, so just @register should work.